wrapper.sh

```diff
@@ -2,10 +2,6 @@
 #set -x
 #set -e
 
-# Sets locale to utf-8 instead of posix. 
```

Comment:
I think we should leave this is to avoid future problem

Reply:
changed back